kafka/test_kafka.py

```diff
@@ -9,7 +10,15 @@
 class KafkaTestCase(DataprocTestCase):
     COMPONENT = 'kafka'
     INIT_ACTIONS = ['kafka/kafka.sh']
+    KAFKA_CRUISE_CONTROL_INIT_ACTION = ['kafka/kafka.sh', 'kafka/cruise-control.sh']
+    KAFKA_MANAGER_INIT_ACTION = ['kafka/kafka.sh', 'kafka/kafka-manager.sh']
     TEST_SCRIPT_FILE_NAME = 'validate.sh'
+    KAFKA_PYTHON_INIT_ACTION_PATH = 'gs://dataproc-kafka-test/kafka_python.sh'
+    KAFKA_PYTHON3_INIT_ACTION_PATH = 'gs://dataproc-kafka-test/kafka_python3.sh'
+    PYTHON2_VERSION = 'python2.7'
+    PYTHON3_VERSION = 'python3.10'
```

Comment:
3.10 might not be the right version for all Dataproc versions. I think we should use `python3` here.

Reply:
done